src/Go/boardAnalysis/goAI.ts

```diff
@@ -38,7 +38,7 @@ export function makeAIMove(boardState: BoardState): Promise<Play> {
         passTurn(boardState, GoColor.white);
         // if passTurn called endGoGame, or the player has no valid moves left, the move should be shown as a game over
         if (boardState.previousPlayer === null || !getAllValidMoves(boardState, GoColor.black).length) {
-          return { type: GoPlayType.gameOver, x: null, y: null };
+          return Promise.resolve({ type: GoPlayType.gameOver, x: null, y: null });
```

Comment:
Promise.resolve is unneeded here - the inner closure here is `asyc`, so all return values are wrapped in a promise if they are not already promises.